genref/config.yaml

```diff
@@ -56,90 +61,108 @@ apis:
       - k8s.io/controller-manager/config/v1alpha1
 
   - name: kube-proxy-config
+    title: kube-proxy configuration (v1alpha1)
     package: k8s.io/kube-proxy
     path: config/v1alpha1
 
   - name: metrics
+    title: ""
     package: k8s.io/metrics
     path: pkg/apis/metrics/v1alpha1
 
   - name: metrics
+    title: ""
     package: k8s.io/metrics
     path: pkg/apis/metrics/v1beta1
 
   - name: client-authentication
+    title: Client authentication (v1alpha1)
     package: k8s.io/client-go
     path: pkg/apis/clientauthentication/v1alpha1
 
   - name: client-authentication
+    title: Client authentication (v1beta1)
     package: k8s.io/client-go
     path: pkg/apis/clientauthentication/v1beta1
 
   - name: apiserver-config
+    title: kube-apiserver configuration (v1alpha1)
     package: k8s.io/apiserver
     path: pkg/apis/apiserver/v1alpha1
     # skipped in favor of v1beta1 and v1
     skip: true
 
   - name: apiserver-config
+    title: kube-apiserver configuration (v1beta1)
     package: k8s.io/apiserver
     path: pkg/apis/apiserver/v1beta1
 
   - name: apiserver-config
+    title: kube-apiserver configuration (v1)
     package: k8s.io/apiserver
     path: pkg/apis/apiserver/v1
 
   # Note: This is named as apiserver-encryption because the so called config
   # pkg only contains encryption config. Need to differentiate it from the
   # apiserver-config above.
   - name: apiserver-encryption
+    title: kube-apiserver encryption configuration (v1)
     package: k8s.io/apiserver
     path: pkg/apis/config/v1
 
     # Note: This is named apiserver-audit because it is part of the apiserver module
   - name: apiserver-audit
+    title: kube-apiserver audit configuration (v1)
```

Comment:
Hi @tengqm .
Is this new field the page (H1 level heading) title?
I assume readers can match the package and path to the title.
The generated page (https://deploy-preview-26606--kubernetes-io-master-staging.netlify.app/docs/reference/config-api/apiserver-audit.v1/#audit-k8s-io-v1-Level) uses the "DisplayName"?

Reply:
Yes, the `title` field will be served as the "title".
The generated pages are not yet updated.